### PR TITLE
Time fix

### PR DESCRIPTION
**Pull request by ethanlchristensen**

### CODE REVIEW

File: BOT/_slash/pause.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            await interaction.response.defer()
+            if user_channel:
+                # SEE IF THE BOT IS IN THE CHANNEL
+                if voice_channel:
+                    # If the bot is currently playing music
+                    if voice_channel.is_playing():
+                        # This is BROKEN
+                        voice_channel.pause()
+                        await interaction.followup.send(
+                            content="Paused!"
```
Comment (nulzo) [inline BOT/_slash/pause.py:30]: please make a verbose embed reply following the standards shown in the "Play", "skip", and "Now Playing" commands.

File: BOT/_slash/pause.py
Comment (ethanlchristensen) [inline BOT/_slash/pause.py:1]: verbose embds coming in next pr

File: BOT/_slash/playing.py [hunk 2]
```diff
@@ -24,21 +27,62 @@ async def playing(interaction: discord.Interaction):
                     "BRUH I AINT EVEN *IN* A VC RIGHT NOW! :skull: :rofl:"
                 )
             # SEE IF THE BOT IS IN THE CHANNEL - WE DON'T NEED TO JOIN
-            elif interaction.guild.voice_client:
+            elif vcli := interaction.guild.voice_client:
```
Comment (nulzo) [inline BOT/_slash/playing.py:30]: no.

File: BOT/_slash/playing.py [hunk 3]
```diff
@@ -24,21 +27,62 @@ async def playing(interaction: discord.Interaction):
...
+
+                            current_time = time()
+
+                            elasped = int(current_time - playing_info.get("started_at"))
+
+                            hours, minutes, seconds = seconds_to_hms(elasped)
+
+                            streaming_for = "${hours} ${minutes} ${seconds}"
+
+                            if hours > 0:
```
[the remark this answers is not in the document]
Comment (nulzo) [reply BOT/_slash/playing.py:46]: Datetime module staring at you from across the room

File: BOT/_utils/boot.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from bruhanimate.bruhscreen import Screen
+from bruhanimate.bruhrenderer import *
+import bruhanimate.images as images
+
+def boot(screen):
+    image = images.text_to_image("F N B B\nD E V S", padding_top_bottom=1, padding_left_right=3)
+    renderer = FocusRenderer(screen, 400, 0, image, "plasma", " ", transparent=False, start_frame=30, reverse=True, start_reverse=220)
```
Comment (nulzo) [inline BOT/_utils/boot.py:7]: pretty flippin le erm epic

File: BOT/_utils/time_utils.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+def seconds_to_hms(seconds: int):
```
Comment (nulzo) [inline BOT/_utils/time_utils.py:1]: https://docs.python.org/3/library/datetime.html

File: BOT/fortnite_balls.py [hunk 6]
```diff
@@ -8,16 +8,16 @@
 """
 
 import re
+import os
```
Comment (nulzo) [inline BOT/fortnite_balls.py]: remove

File: BOT/fortnite_balls.py [hunk 7]
```diff
@@ -8,16 +8,16 @@
...
+import time
```
Comment (nulzo) [inline BOT/fortnite_balls.py]: remove

File: BOT/_slash/pause.py
Comment (nulzo) [reply BOT/_slash/pause.py:1]: image

**Review by nulzo — APPROVED** [review]

**Pull request merged**